spec/features/trusted_login_flow_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+feature 'Sign in using trusted parameters', js: true do
+  background do
+    load 'db/seeds.rb'
+    create_default_application
+  end
+  let(:role) { 'instructor' }
+  let(:payload) {
+    {
+      role:  role,
+      external_user_uuid: SecureRandom.uuid,
+      name:  'Tester McTesterson',
+      email: 'test@test.com',
+      school: 'Testing U'
+    }
+  }
+  let(:signed_params) {
+    { sp: OpenStax::Api::Params.sign(params: payload, secret: @app.secret) }
+  }
+
+  describe 'instructors' do
```

Comment:
I think we need a spec to show a returning LMS user gets logged in immediately? (probably don't need instructor and student variants)

Reply:
yep, added that and one for the terms still showing up